seaway.API/Scripts/25052024.sw.sql

```diff
@@ -0,0 +1,86 @@
+IF EXISTS (SELECT 1 FROM sysobjects WHERE id = object_id(N'[dbo].[DeleteActivity]') AND OBJECTPROPERTY(id, N'IsProcedure') = 1)
+BEGIN
+	DROP PROCEDURE DeleteActivity
+END
+GO
+
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+
+-- =============================================
+-- Author:		Kavishan
+-- Create date: 24/05/2024
+-- Description:	Delete Activity
+-- =============================================
+CREATE PROCEDURE [dbo].DeleteActivity
+	@activityId INT
+AS
+BEGIN
+	SET NOCOUNT ON;
+	IF EXISTS(SELECT 1 FROM Activities WHERE ActivityId = @activityId)
+	BEGIN
+		DELETE FROM Activities WHERE ActivityId = @activityId
+		IF EXISTS(SELECT 1 FROM PicDocuments WHERE PicTypeId = @activityId AND PicType = 'Activity')
+			BEGIN
+				DELETE FROM PicDocuments WHERE PicTypeId = @activityId AND PicType = 'Activity'
+			END
+	END
+END
+GO
+
+
+
+
+
+
+USE [seaway]
```

Comment:
remove this use [seaway] comment 
and put the if condition for checking the SP existings